gptel-transient.el

```diff
@@ -752,7 +752,26 @@ Customize `gptel-directives' for task-specific prompts."
     :pad-keys t])
 
 ;; ** Prefix for selecting tools
+(defun gptel--toggle-all-tools ()
```

Comment:
Move this function to the "Helper functions and vars" section, to around line 385.

Reply:
done.